internal/service/common/utils/repository.go

```diff
@@ -101,8 +101,6 @@ func Delete[T db.Model](ctx context.Context, db DBQuery, whereExpr psql.Expressi
 		return 0, fmt.Errorf("failed to build delete query for '%s': %w", record.TableName(), err)
 	}
 
-	slog.Debug("executing statement", "sql", sql, "args", args)
```

Comment:
I suppose you are deleting this because the bulk operations have a lot of values. However, by experience, it is good to have visibility to what we send to the DB. We will need them.

Reply:
yeah this was interesting issue. not sure if you tried this in the cluster...but basically after generating these large logs...slog log package effectively breaks. Like the server would start but no more logs are shown after a certain point. I was able to see the logs in crictl...but not on the console. 

yeah the logs are giant. like one call to upload the defs is basically few pages long.

Reply:
I didn't see that issue. Do you also have it without the refactoring of the dicts you are implementing here?

Reply:
Yeah that was the main reason for refactoring actually. Originally i thought errors were not currently bubbling up and things werent working for me. So i started splitting it up see if there any issues there...but yeah turns out that crictl was showing the logs but app was basically still.

Reply:
and what was the issue?